tests/jest.config.unit.ts

```diff
@@ -126,6 +128,8 @@ export default {
 	roots: [
 		'<rootDir>/taqueria-sdk',
 		'<rootDir>/tests/unit/taqueria-sdk',
+		'<rootDir>/taqueria-plugin-ligo',
+		'<rootDir>/tests/unit/ligo-plugin-tests',
```

Comment:
we can delete this (`'<rootDir>/tests/unit/ligo-plugin-tests'`) right?

Reply:
jest needs to look there for files. test won't get run without that